liquibase-standard/src/main/java/liquibase/analytics/LiquibaseAnalyticsListener.java

```diff
@@ -78,18 +78,18 @@ public void handleEvent(Event event) throws Exception {
                 Yaml yaml = new Yaml(dumperOptions);
                 yaml.setBeanAccess(BeanAccess.FIELD);
 
-                SegmentBatch segmentBatch = SegmentBatch.fromLiquibaseEvent(event, userId);
-                String jsonInputString = YamlSerializer.removeClassTypeMarksFromSerializedJson(yaml.dumpAs(segmentBatch, Tag.MAP, DumperOptions.FlowStyle.FLOW));
+                AnalyticsBatch analyticsBatch = AnalyticsBatch.fromLiquibaseEvent(event, userId);
+                String jsonInputString = YamlSerializer.removeClassTypeMarksFromSerializedJson(yaml.dumpAs(analyticsBatch, Tag.MAP, DumperOptions.FlowStyle.FLOW));
                 // This log message is purposefully being logged at fine level, not using the configurable log-level param, so that users always know what is being sent to Segment.
-                logger.log(logLevel, "Sending analytics to Segment. " + segmentBatch, null);
+                logger.log(logLevel, "Sending anonymous data to Liquibase analytics endpoint. " + analyticsBatch, null);
```

Comment:
Just curious, what does it mean "anonymous data"?

Reply:
@MalloD12 means we are not identifying the user that is sending the data. We are also adding the message so users don't worry about the data being associated with them.